hilfe/handbuch/mitarbeiter/vertrag/index.md

```diff
@@ -164,13 +167,15 @@ Wird ein Mitarbeiter **nach Stunden** bezahlt, wählen Sie im Dropdown-Menü **S
 
 Wenn ein Mitarbeiter **Umsatzprovision** erhält, geben Sie unter Provision den Prozentsatz ein. **Voraussetzung** für die Berechnung der Provision ist die **Erfassung** des individuellen **Umsatzes** im Hauptmenüpunkt "[**Zeiterfassung**](/hilfe/handbuch/zeiterfassung/tagesuebersicht/#provision)".
 
+--
 Wenn sich die Provision auf den Netto-Umsatz bezieht, ist dort der Netto-Umsatz des Mitarbeiters einzugeben. Bezieht sie sich auf den Brutto-Umsatz, entsprechend der Brutto-Umsatz.
```

Comment:
"Wenn sich die Provision auf den Netto-Umsatz bezieht, ist dort der Netto-Umsatz des Mitarbeiters einzugeben. Bezieht sie sich auf den Brutto-Umsatz, entsprechend der Brutto-Umsatz."
Ist das ein Relikt aus einer älteren Version? Ich werde daraus nicht wirklich schlau, da man bei den Verträgen ja keine Umsätze eingibt - erschien mir aber wichtig.

Reply:
Abseits von diesem Punkt ist alles mergebereit

Reply:
Grade nochmal im Kontext des Artikels gelesen und in dem Kontext ist der Passus tatsächlich eher verwirrend. Hab ihn rausgenommen.